### DISCUSSION TITLE
etcd backup monitoring and alerting

### DISCUSSION THREAD

**Discussion by dracon80** [k3s-io/k3s#9038] [Q&A]
I have setup prometheus and I'm getting all the stats on my embedded etcd datastore scraped and into prometheus, along with grafana dashboard. What I can't figure out how to setup is a simple alert to notify me that backups of etcd have failed. I don't currently see anything in the metrics that would allow me to do this.
Am I just missing the obvious? or do I need to look at alternatives to get alerts on this backup process?

**Comment by brandond** [comment]
There are not currently any Prometheus metrics for the built in snapshot functionality.

**Comment by dracon80** [comment]
Any suggestions on ways to monitor the backup status?
